### ISSUE TITLE
svo node not publishing any results using stereo+imu camera

### ISSUE THREAD

**Issue by david-delmoral** [uzh-rpg/rpg_svo_pro_open#45]
I am attempting svo with a realsense t265, but when I launch the node with a launch file similar to the fla_stere_imu.launch, Rviz opens up but nothing is displayed, there is no error message, although the svo node in not publishing data in any of the topics. 

The step I've followed are:
- Since the camera publishes the IMU data segregated on different topics, I have made a python node that mixes them in a single IMU message. 
- In the launch file, I set the camera topics to the ones published by the t265, and the IMU topic to the one published by the python node.
- For the calibration file I used the format of the fla_stereo_imu.yaml introducing the distortion values and intrinsics published by the t265 at /camera/fisheye1/camera_info and /camera/fisheye2/camera_info.
- For the parameter file I used the same as in fla_stere_imu.launch
- I have tried both running from a bag file with the three topics recorded, and running the camera node live and in either case I get the same result.
In rqt I get the following chart:
![image](https://user-images.githubusercontent.com/126670830/224319904-53f53121-e97c-4c5e-925a-7bb0a4ba72dc.png)
where you can see that the svo node is subscribed to the camera and IMU topics, but none of the topics between svo and vis contain any data.
I don't know if I have missed any steps needed to use svo with a custom stereo + imu camera.
Thank you for the help

**Comment by zauberflote1** [comment]
@david-delmoral Did you find a solution?

**Comment by david-delmoral** [comment]
> @david-delmoral Did you find a solution?

No. I ended up using other algorithms for my proyect bc I couldn't get svo working

**Comment by MauUnterKarottenbaum** [comment]
same... waiting for further tips

**Comment by aatb-ch** [comment]
i also stumbled upon the same issue, running fine in mono but no results in stereo, the stereoCallback in svo_interface expects exact timestamps for both camera streams, if you switch the sync policy from ExactTime to ApproximateTime it works fine. My camera (oak-d pro w) outputs slightly different timestamps for both streams so no result was published, as soon as I switched to ApproximateTime and recompiled it started working fine.

https://github.com/uzh-rpg/rpg_svo_pro_open/blob/ca371f304637e7fb355cf4624d0a02da4e3da220/svo_ros/src/svo_interface.cpp#L504
